docs/resolver-template/cmd/demoresolver/main.go

```diff
@@ -17,7 +17,7 @@ import (
 	"context"
 	"errors"
 
-	pipelinev1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
```

Comment:
NIT: I suppose these changes could have been done before in their dedicated PR

Reply:
I think this is related to https://github.com/tektoncd/pipeline/pull/6444/files#r1227105161 and we might want to try to swap all the v1beta1 params for resolver to v1 in the same PR since the reconciler previously uses v1beta1 fields/CRDs until this PR?